packages/contracts-bedrock/src/dispute/interfaces/IAnchorStateRegistry.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
```

Comment:
this pragma should be `^0.8.0`

Reply:
Looks like we need to fix all of our interfaces, good catch. Updating.